geospatial/dted.ksy

```diff
@@ -0,0 +1,596 @@
+          text or zero filled)).
+      - id: reserved_for_future_use2
+        type: str
+        size: 8
+        doc: |
+          Reserved for future use. (Blank filled).
+      - id: data_edition_number
+        type: str
+        size: 2
+        doc: |
+          Data Edition Number (01-99).
+      - id: match_merge_version
+        type: str
+        size: 1
+        doc: |
+          Match/Merge Version (A-Z).
+      - id: maintenance_date
+        type: date_yymm
+        doc: |
+          Match/Merge Date. Zero filled until used. (YYMM)
+      - id: match_merge_date
+        type: date_yymm
+        doc: |
+          Match/Merge Date. Zero filled until used. (YYMM)
+      - id: maintenance_description
+        type: str
+        size: 4
+        doc: |
+          Maintenance Description Code. Zero filled until used. (0000 or ANNN)
+      - id: producer_code
+        type: str
+        size: 8
+        doc: |
+          Producer Code.(Country - Free Text) (FIPS 10-4 Country Codes used 
+          for first 2 characters) (CCAAABBB).
+      - id: reserved_for_future_use3
+        type: str
+        size: 16
+        doc: |
+          Reserved for future use. (Blank filled).
+      - id: product_specification
+        type: str
+        size: 9
+        doc: |
+          Product Specification. (Alphanumeric field) (AAAAAAAAA)
+      - id:  product_specification_number
+        type: str
+        size: 2
+        doc: |
+          First digit is Product Specification Amendment Number and second |
+          digit is the Change Number.
+      - id:  product_specification_date
+        type: date_yymm
+        doc: |
+          Date of Product Specification.
+      - id:  vertical_datum
+        type: str
+        size: 3
+        doc: |
+          Vertical Datum (MSL, E96)
+      - id:  horizontal_datum_code
+        type: str
+        size: 5
+        doc: |
+          Horizontal Datum Code (Current Version World Geodetic System) (WGS84).
+      - id:  digitizing_collection_system
+        type: str
+        size: 10
+        doc: |
+          Digitizing/Collection System. (Free text).
+      - id:  compilation_date
+        type: date_yymm
+        doc: |
+          Compilation Date. (Most descriptive year/month) (YYMM).
+      - id:  reserved_for_future_use4
+        type: str
+        size: 22
+        doc: |
+          Reserved for future use. (Blank filled).
+      - id:  origin_of_data
+        type: coordinate_pair_ddmmss_sh
+        doc: |
+          Latitude of origin of data— leading zero for values less than 10; H |
+          is the hemisphere of the data (DDMMSS.SH). |
+          Longitude of origin of data— leading zeroes for values less than |
+          100; H is the hemisphere of the data (DDDMMSS.SH).
+      - id:  sw_corner_of_data
+        type: coordinate_pair_ddmmssh
+        doc: |
+          Latitude of SW corner of data, bounding rectangle— leading zero for |
```

Comment:
I wonder if part of this docs should be moved into type doc.

Reply:
That is a good idea. The documentation will be different than what is in the spec but since I chose to  add this type instead of repeating my self updating the documentation to reflect that I think would be a good idea.